raiden-cli/src/config.json

```diff
@@ -4,6 +4,20 @@
   "autoSettle": true,
   "autoUDCWithdraw": false,
   "caps": {
-    "Receive": 1
+    "Receive": 1,
+    "Mediate": 1,
+  },
+  "mediationFees": {
+    "0x0000000000000000000000000000000000000000": {
+      "flat": 0, 
+      "prop": 4000,
+      "inbalance": 3000,
+    },
+    "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2": {
+      "flat": 10000000000,
```

Comment:
Doesn't Adnre say here that this will reset the proportional and imbalance fees to zero? https://github.com/raiden-network/light-client/issues/2790#issuecomment-856891739

Reply:
Ah and the PC doesn't? Sorry then I misunderstood you when I asked you. Sure then this needs adoption to set them to zero.

Reply:
Thanks for the clarifying talk. Sorry that I messed this up. 😑